_pages/Home/components/Login/index.js

```diff
@@ -1,22 +1,41 @@
 import { Formik, Field, Form } from 'formik';
 import styles from './styles/login.module.css';
+import axios from 'axios';
+import Cookies from 'universal-cookie';
+import { useRouter } from 'next/router';
+
+const cookies = new Cookies();
 
 export function LoginForm() {
+  const router = useRouter();
   return (
     <div className={styles.form}>
       <h1>Login</h1>
       <Formik
-        initialValues={{ email: '', password: '' }}
+        initialValues={{ email: 'stephen-1310@hotmail.com', password: 'Mag3.Rul35' }}
         onSubmit={async (values) => {
-          await new Promise((resolve) => setTimeout(resolve, 500));
-          alert(JSON.stringify(values, null, 2));
+          // await new Promise((resolve) => setTimeout(resolve, 500));
```

Comment:
Do you still want/need these comments? If not, it's good hygiene to merge commits without any unneeded comments, console.logs, etc.